backend-api/src/functions/asyncBiometricToken/asyncBiometricTokenHandler.test.ts

```diff
@@ -1,27 +1,119 @@
-import { Context } from "aws-lambda";
-import { Logger } from "../services/logging/logger";
-import { MockLoggingAdapter } from "../services/logging/tests/mockLogger";
+import { expect } from "@jest/globals";
+import "../testUtils/matchers";
+import "dotenv/config";
+import { APIGatewayProxyResult, Context } from "aws-lambda";
 import { buildLambdaContext } from "../testUtils/mockContext";
 import { buildRequest } from "../testUtils/mockRequest";
 import { lambdaHandlerConstructor } from "./asyncBiometricTokenHandler";
 import { IAsyncBiometricTokenDependencies } from "./handlerDependencies";
-import { MessageName, registeredLogs } from "./registeredLogs";
 import {
   expectedSecurityHeaders,
   mockSessionId,
 } from "../testUtils/unitTestData";
+import { logger } from "../common/logging/logger";
+import { emptyFailure, successResult } from "../utils/result";
 
 describe("Async Biometric Token", () => {
   let dependencies: IAsyncBiometricTokenDependencies;
   let context: Context;
-  let mockLoggingAdapter: MockLoggingAdapter<MessageName>;
+  let consoleInfoSpy: jest.SpyInstance;
+  let consoleErrorSpy: jest.SpyInstance;
+  let result: APIGatewayProxyResult;
+
+  const validRequest = buildRequest({
+    body: JSON.stringify({
+      sessionId: mockSessionId,
+      documentType: "NFC_PASSPORT",
+    }),
+  });
+
+  const mockSuccessfulGetSecrets = jest
```

Comment:
nit[take it or leave it]:
```suggestion
  const mockGetSecretsSuccess = jest
```

Reply:
Done